Kickstarter-iOS/Library/SharedFunctions.swift

```diff
@@ -1,6 +1,38 @@
 import Library
 import UIKit
 
+// MARK: - Haptic feedback
+
+func generateImpactFeedback(
```

Comment:
Convenience functions should now be unified place to trigger with the following improvements:
- we call `prepare` to get the HW ready for haptic feedback
- we do not store reference to the haptic feedback generator (so the system releases it right after)